content/concepts/concepts.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Concepts
+---
+
+webpack is [incredibly configurable](./api/configuration), however, there are **4 Core Concepts** we feel you should understand before you get started! 
+
+As part of your webpack learning journey, we wrote this document aimed to give you a **high-level** overview of these concepts, while still providing links to concept specific use-cases. 
+
+## Entry
+In the [previous article](./index), we mentioned that webpack creates a graph of all of your application's dependencies. The starting point of this graph is known as an _entry point_. The _Entry point_ tells webpack _where to start_ and follows the graph of dependendencies to know _what to bundle_. You can think of your applications _entry point_ as the **contextual root** or **the first file to kick off your app**.
+
+In webpack we define _entry points_ using the `entry` property in our [webpack configuration object](./configuration). 
+
+The simplest example is seen below: 
+
+**webpack.config.js**
+```
+  module.exports = config; 
+
+  const config = {
+    entry: './path/to/my/entry/file.js' 
+  }
+
+```
+
+There are multiple ways to declare your `entry` property that are specific to your applications needs. 
+
+[**Learn more!**](./entry-points)
+
+## Output
+Once you've bundled all of your assets together, we still need to tell webpack **where** to bundle our application. The webpack `output` property describes to webpack **how to treat bundled code**. 
+
+**webpack.config.js**
+```
+  module.exports = config; 
```

Comment:
I wanted to embed the idea right away, that these configs are code, exporting an object. And that all that matters is the properties inside of that object. So by using a variable, it kind of indirectly reinforces that concept. @jhnns @bebraw if you both vote to export directly I am fine with that.